app/admin/api/edit.php

```diff
@@ -94,7 +94,7 @@
 	    <td>
 	    	<select name="app_security" class="form-control input-sm input-w-auto">
 	    	<?php
-	    	$perms = array("SSL with User token"=>"ssl_token","SSL with App code token"=>"ssl_code","Encrypted"=>"crypt", "User token"=>"none");
+	    	$perms = array( _("SSL with User token")=>"ssl_token", _("SSL with App code token")=>"ssl_code", _("Encrypted")=>"crypt", _("User token")=>"none");
 
 	    	foreach($perms as $k=>$p) {
 		    	if($p==$api->app_security)		{ print "<option value='$p' selected='selected'>"._($k)."</option>"; }
```

Comment:
Please remove the deferred translations as you've already _() the text in the $perms array above,
e.g  first loop: $k = _("SSL with User token") and $p = "ssl_token", then we run $k through _() again.

```php
  print "<option value='$p' selected='selected'>"._($k)."</option>";
```
=>
```php
 print "<option value='$p' selected='selected'>".$k."</option>";
```